packages/ERTP/src/amountMath.js

```diff
@@ -38,44 +46,46 @@ import './types';
  * function `coerce` takes an amount and checks it, returning an amount (amount
  * -> amount).
  *
- * `makeAmount` takes in a brand and the name of the particular
- * mathHelpers to use.
+ * `makeAmountMath` takes in a brand and the kind of mathHelpers to use.
  *
  * amountMath is unfortunately not pass-by-copy. If you call
- * `getAmountMath` on a remote issuer, it will be a remote object and
+ * `getAmountMath` on a remote issuer, it will return a remote object and
  * each call will incur the costs of calling a remote object. However,
  * you can create a local amountMath by importing this module locally
- * and calling makeAmountMath(), passing in a brand and a mathHelpers name,
- * both of which can be passed-by-copy (since there are no calls to brand
- * in this module).
+ * and passing `makeAmountMath` the brand and the kind of amountMath. The
+ * amountMath kind is a pass-by-copy string, while the brand is a presence.
+ * There's no performance cost for the brand being a presence, as the only
+ * operations on the brand in this module are equality tests.
+ *
+ * AmountMath exports amountMathKinds, which contains constants for the kinds:
+ * NAT, SET, and STR_SET.
  *
  * Each issuer of digital assets has an associated brand in a one-to-one
  * mapping. In untrusted contexts, such as in analyzing payments and
  * amounts, we can get the brand and find the issuer which matches the
  * brand. The issuer and the brand mutually validate each other.
  *
  * @param {Brand} brand
- * @param {MathHelpersName} mathHelpersName
+ * @param {AmountMathKind} amountMathKind
```

Comment:
I like the "kind" renaming.

Reply:
The only thing I'm unhappy about with it is the prose comment above, which uses 'kind' to mean both the distinction between assets from different brands and the difference between 'nat' and 'set'.